.github/workflows/test.yml

```diff
@@ -0,0 +1,47 @@
+name: Test
+on: [pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Read .nvmrc
+      run: echo ::set-output name=NVMRC::$(cat .nvmrc)
+      id: nvm
```

Comment:
Is this really the best way to read the nvmrc file? 😱 

Something to look at later I suspect

Reply:
Good point, meant to add clarification around this. There are a number of approaches to replicate what we were doing before with nvm on travis CI, this one I got from https://github.com/actions/setup-node/issues/32#issuecomment-539794249

Reply:
Awesome, clarification would be good

Reply:
@lfdebrux Scratch that it appears from later comments nvm got bundled so there may be a nicer approach closer to what we do in the makefile I will take a look.

Reply:
@lfdebrux thanks for raising this and making me look again at it. I've changed it now to use `nvm` directly in ab5307c.

Reply:
Cool!